.gitignore

```diff
@@ -35,8 +35,3 @@ plans/**/go.sum
 .history
 
 # End of https://www.gitignore.io/api/go,visualstudiocode
-
-## tarballs
```

Comment:
This is rather dangerous - if you have other .tgz charts in `infra/k8s/testground-infra/Charts`, they will be installed.

This resulted in me installing `prometheus` twice, without realising.

Therefore I am removing the `*.tgz` gitignore, so that folks at least maintain a clean working directory, when running the `install.sh` script.

Reply:
@coryschwartz we should not be downloading charts in the working dir of `testground` I think.

We should also start to properly version our charts - for example `testground-infra` chart 0.0.1 had prometheus as dep, and now no longer has it.

If it is not for this .tgz ignore, users who have used Testground in a previous release will have a hard time installing this release.